tests/integration/data/legacy/case-07.md

```diff
@@ -0,0 +1 @@
+# About case 7
```

Comment:
Do you want to say more?

Reply:
I was copying tests from a python file to json files, and transferred comments to a companion markdown file. Not all of the json-in-python had comments. I'll inspect those tests and add relevant comments.